.rubocop_todo.yml

```diff
@@ -1,22 +1,11 @@
 # This configuration was generated by
-# `rubocop --auto-gen-config --no-auto-gen-timestamp`
-# using RuboCop version 1.74.0.
+# `rubocop --auto-gen-config`
```

Comment:
There is a `bin/rubocop-todo` script

Reply:
Ah i didn't realise, thanks. I have used that now.